capa/render/result_document.py

```diff
@@ -22,6 +24,13 @@
 from capa.engine import MatchResults
 from capa.helpers import assert_never
 
+try:
+    from functools import lru_cache
+except ImportError:
+    # need to type ignore this due to mypy bug here (duplicate name):
+    # https://github.com/python/mypy/issues/1153
+    from backports.functools_lru_cache import lru_cache  # type: ignore
+
```

Comment:
```suggestion
from functools import lru_cache

```

`functools.lru_cache` was added in python3.2, which is older than our minimum support python version, so we can assume it exists, and no need to guard the import.